src/configs/phenotype_config.py

```diff
@@ -41,8 +41,7 @@
 
 def get_accuracy(y_true: np.array, y_pred: torch.tensor) -> float:
     """ Takes predictions, gets most probable class and compares with y to get accuracy """
-    return int(torch.sum(torch.max(torch.tensor(y_pred), 1).indices == torch.from_numpy(y_true))) / len(y_true)
-
+    return int(torch.sum(torch.max(torch.tensor(y_pred), 1).indices == y_true)) / len(y_true)
```

Comment:
Why we receive true values as numpy array and preds as torch tensor? We should still have y_true as torch.tensor somewhere. Also, why this function is in phenotype_config?